packages/infra/lib/surescripts-stack.ts

```diff
@@ -0,0 +1,527 @@
+import { Duration, NestedStack, NestedStackProps } from "aws-cdk-lib";
+import { SnsAction } from "aws-cdk-lib/aws-cloudwatch-actions";
+import * as ec2 from "aws-cdk-lib/aws-ec2";
+import { Function as Lambda } from "aws-cdk-lib/aws-lambda";
+import { SqsEventSource } from "aws-cdk-lib/aws-lambda-event-sources";
+import * as s3 from "aws-cdk-lib/aws-s3";
+import { Queue } from "aws-cdk-lib/aws-sqs";
+import { Construct } from "constructs";
+import { EnvConfig } from "../config/env-config";
+import { EnvType } from "./env-type";
+import { createLambda } from "./shared/lambda";
+import { LambdaLayers } from "./shared/lambda-layers";
+import { LambdaSettings, QueueAndLambdaSettings } from "./shared/settings";
+import { createQueue } from "./shared/sqs";
+
+const connectSftpTimeout = Duration.seconds(30);
+const synchronizeSftpTimeout = Duration.minutes(15);
+const sendPatientRequestLambdaTimeout = Duration.minutes(12);
+const receiveVerificationLambdaTimeout = Duration.minutes(5);
+const receiveFlatFileResponseLambdaTimeout = Duration.minutes(15);
+
+interface SurescriptsSettings {
+  connectSftp: LambdaSettings;
+  synchronizeSftp: QueueAndLambdaSettings;
+  sendPatientRequest: QueueAndLambdaSettings;
+  receiveVerificationResponse: QueueAndLambdaSettings;
+  receiveFlatFileResponse: QueueAndLambdaSettings;
+}
+
+const settings: SurescriptsSettings = {
+  connectSftp: {
```

Comment:
Why do we need a connect SFTP lambda separated from the synchronize one? Is there a DD/plan for this?

Reply:
This is just for testing the Surescripts connection, like a ping. Actually would make sense to just merge it into a single synchronization Lambda for simplicity.